jmix-core/core/src/main/java/io/jmix/core/impl/FileTransferServiceImpl.java

```diff
@@ -202,6 +204,12 @@ public ResponseEntity<FileInfoResponse> fileUpload(String name,
             } catch (NumberFormatException ignored) {
             }
 
+            if (size > coreProperties.getMaxFsFileSize().toBytes()) {
```

Comment:
Checking Content-Length header is not reliable and cannot prevent a deliberate attack. So I'm afraid the whole idea with CoreProperties.maxFsFileSize is useless.

Reply:
Removed, max file size moved to  LocalFileStorageProperties